server/chat.ts

```diff
@@ -2016,7 +2014,7 @@ export const Chat = new class {
 		return `<div class="message"><ul class="utilichart">${buf}<li style="clear:both"></li></ul></div>`;
 	}
 	getDataMoveHTML(move: Move) {
-		if (typeof move === 'string') move = Object.assign({}, Dex.getMove(move));
+		if (typeof move === 'string') move = Dex.getMove(move);
```

Comment:
@Spandamn Do you remember why you cloned these? It seems unnecessary.

Reply:
Since `move` can't even be a string these days, the whole line seems unnecessary.